index-de.html

```diff
@@ -195,7 +200,7 @@ <h3 id="commit">Commit</h3>
   <p>Ein vierter Fall ist, ein Commit mit <a href="#detached">detached HEAD</a>,
   wird weiter unten erklärt.</p>
 
-  <h3 id="checkout">Checkout</h3>
+  <h3 id="checkout">Checkout (Ausprobieren)</h3>
```

Comment:
If l understand [that](http://stackoverflow.com/questions/5244786/why-is-the-git-command-to-switch-branches-named-git-checkout) correctly, "checkout" is rather metaphorical (see Davis' comment).
So you could call it ```(Ausstempeln)```. (l found this one [at dict.cc](http://www.dict.cc/?s=check+out))